Makefile

```diff
@@ -31,4 +31,10 @@ build:
 	operator-sdk bundle create --directory . $(OPERATOR_REPOSITORY)/$(OPERATOR_IMAGE_NAME)-bundle:$(OPERATOR_BUNDLE_VERSION)
 	docker push $(OPERATOR_REPOSITORY)/$(OPERATOR_IMAGE_NAME)-bundle:$(OPERATOR_BUNDLE_VERSION)
 	opm index add --bundles  $(OPERATOR_REPOSITORY)/$(OPERATOR_IMAGE_NAME)-bundle:$(OPERATOR_BUNDLE_VERSION) --tag $(OPERATOR_REPOSITORY)/$(OPERATOR_IMAGE_NAME)-index:$(OPERATOR_BUNDLE_VERSION) --build-tool=docker
-	docker push $(OPERATOR_REPOSITORY)/$(OPERATOR_IMAGE_NAME)-index:$(OPERATOR_BUNDLE_VERSION)
\ No newline at end of file
+	docker push $(OPERATOR_REPOSITORY)/$(OPERATOR_IMAGE_NAME)-index:$(OPERATOR_BUNDLE_VERSION)
+
+.PHONY: test-bundle-install
+## Runs smoke tests to verify the bundle install
+test-bundle-install:
+	@echo "Testing the new operator bundle install" 
+	./setup-test-env.sh
```

Comment:
call the script `test-bundle-install.sh` that would match better.

and add a new line after 40; sometimes makefile are sensitive to this. (github complains actually)

Reply:
user $(Q) instead of ./ also